### PR TITLE
Clarify master branch purpose

### PR DESCRIPTION
**Pull request by pointlessone**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,18 @@ file][config].
 
 [config]: http://eslint.org/docs/user-guide/configuring#using-configuration-files
 
+### Channels
+
+There are a few major versions of ESLint out there. The latest major version
+is 4. This engine provides a few channels for major versions of ESLint.
+
+This branch is for the `latest` (i.e. default) channel provides ESLint 3
```
Comment (maxjacobson) [inline README.md]: small grammar nit: please change `channel provides` to `channel and provides`

File: README.md [hunk 2]
```diff
@@ -15,6 +15,18 @@ file][config].
...
+support.
+
+For ESLint 4 please see [`channels/eslint-4`][] branch.
```
Comment (maxjacobson) [inline README.md]: small grammar nit: please change this to: `For ESLint 4, please see the [channels/eslint-4][] branch.`

File: README.md [hunk 3]
```diff
@@ -15,6 +15,18 @@ file][config].
 
 [config]: http://eslint.org/docs/user-guide/configuring#using-configuration-files
 
+### Channels
+
+There are a few major versions of ESLint out there. The latest major version
+is 4. This engine provides a few channels for major versions of ESLint.
```
Comment (maxjacobson) [inline README.md]: will you make "channels" link to https://docs.codeclimate.com/docs/engine-channels please?

File: README.md [hunk 4]
```diff
@@ -15,6 +15,19 @@ file][config].
...
+There are a few major versions of ESLint out there. The latest major version
+is 4. This engine provides a few [channels][] for major versions of ESLint.
+
+This branch is for the `latest` (i.e. default) channel and provides ESLint 3
+support.
+
+For ESLint 4, please see [`channels/eslint-4`][] branch.
+
+[channels]: https://docs.codeclimate.com/docs/engine-channels
+[`channels/eslint-4`]: https://github.com/codeclimate/codeclimate-eslint/tree/channel/eslint-4
```
Comment (dblandin) [inline README.md]: Small typo here. The branch name is `channel/eslint-4`.

Comment (pointlessone) [inline README.md]: Thanks!

**Pull request merged**